karma.debug.conf.js

```diff
@@ -0,0 +1,16 @@
+// Karma configuration
+
+module.exports = function (config) {
+    'use strict';
+
+    var commonConfig = require('./karma.common.conf');
+
+    commonConfig(config, {
+        singleRun: false,
+        autoWatch: true,
+        browsers: ['Chrome'],
```

Comment:
It's actually pretty easy to run with Firefox, using [karma-runner/karma-firefox-launcher](https://github.com/karma-runner/karma-firefox-launcher). I'll see about running with Firefox in Travis (and maybe nightly, Aurora, etc.).